nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -23,6 +23,12 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         <XamlNamespace Condition="'$(XamlNamespace)' == ''">Windows.UI.Xaml</XamlNamespace>
         <XamlMetaDataProviderIdl Condition="'$(XamlMetaDataProviderIdl)'== ''">$(GeneratedFilesDir)XamlMetaDataProvider.idl</XamlMetaDataProviderIdl>
         <XamlMetaDataProviderCpp Condition="'$(XamlMetaDataProviderCpp)'== ''">$(GeneratedFilesDir)XamlMetaDataProvider.cpp</XamlMetaDataProviderCpp>
+
+        <!-- For CX projects, CppWinRT will never output a winmd-->
+        <!-- NOTE: We don't set a default here as the default requires evaluation of project references
+             and this is done by the ComputeCppWinRTResolvedWinMD target. -->
+        <CppWinRTGenerateWindowsMetadata Condition="'$(CppWinRTGenerateWindowsMetadata)' == '' AND '$(CppWinRTProjectLanguage)' == 'C++/CX' ">false</CppWinRTGenerateWindowsMetadata>
```

Comment:
this line is redundant, same with the pair of checks below (line 33) - checking XamlLanguage == 'C++' should suffice